src/app/components/action-button/index.html

```diff
@@ -31,4 +31,162 @@
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-on-workspace-dont.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Do not use actions buttons when the consequential differences between the choices are a small minority of the scale of the task, or when the choice would not be made at the beginning of the task. Use a different selection input and progressive disclosure to reveal additional content, if needed.
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-inside-long-dialogue-dont.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <header><div class="sky-section-heading">Workflow placement and behaviors</div></header>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Use a set of action buttons inside a modal when no page already exists, or the remaining bulk of the task after the action button decision takes place inside a modal. This is common when creating a new record that only requires a standard modal during different versions of its 'Add' process.
+            </p>
+            <ul>
+                <li>Use a 'default' size modal for 2 action buttons and a 'largeModal' for 3 or more.</li>
+                <ul>
+                    <li>Center the action buttons inside the modal, wrapping to another row if needed.</li>
+                </ul>
+                <li>When the user selects a choice, dismiss this modal and open whatever container is appropriate for the remainder of the task.</li>
+                <ul>
+                    <li>If the user cancels or abandons the task afterwards, do not return the user to this modal.</li>
+                </ul>
+                <li>Always provide a "Cancel" action inside a modal with action buttons.</li>
+            </ul>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-in-modal.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            When the user is completing an Add or configuration process contained inside a page or tab (that includes no other task), use action buttons directly on the page. Most 'Add' processes will use the modal example above.
```

Comment:
I just worry about the pronoun "they" in that suggestion - people might interpret the antecedent to be "action buttons" instead of "pages".

What about "Use action buttons directly on pages that include add or configuration processes, but no other tasks."